src/CarbonAware.DataSources/CarbonAware.DataSources.Json/src/Configuration/JsonDataConfiguration.cs

```diff
@@ -0,0 +1,49 @@
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace CarbonAware.DataSources.Json.Configuration;
+
+/// <summary>
+/// A configuration class for holding Json Data config values.
+/// </summary>
+public class JsonDataConfiguration
+{
+    private const string BaseDir = "data-files";
```

Comment:
Consider a more specific base directory for this data source.

`/json-data-source-files`
`/json-data-source`
`/data-sources/json` <-- I kind of like this one since it follows the same namespace conventions as the classes themselves.
`/data-sources/json/files`